cypress/integration/dashboard/stats_spec.js

```diff
@@ -0,0 +1,52 @@
+import homePage from '../../pages/home';
+
+describe('Homepage stats', () => {
+  beforeEach(() => {
+    cy.server();
+    cy.route('POST', '/api/2.0/dojos/users', [{ userTypes: ['champion'] }]).as('userDojos');
```

Comment:
Should include permissions? Stats would be visible to mentors with dojo-admin rights, wouldn't they?

Reply:
ticketing admin rather ?

Reply:
Only Dojo admin gives you access to the users of a Dojo though, right?

Reply:
Arrgh, you're right. So we can bin the mentors from that pie chart cc @conoramurphy 
I guess at that point, it's more sensible to just say champions @DanielBrierton  ? If we agree the perms are not editable anymore

Reply:
Well, it's one thing to say perms aren't editable anymore (also, not entirely sure I agree just yet since I don't quite know what would replace it), but they are... And probably will be for a while...

So I'd say for now, this needs to check for dojo-admin until we actually make that decision and make the necessary perms changes for it.

Reply:
Sorry just to clarify. Are we saying it's extra work to show it to mentors with dojo-admin rights?

Reply:
Or that we can't seperate the mentors easily? Why would we leave out mentors.

Reply:
Because either we rely on perms, which we dont' want to for deprecation purpose, or we rely on role, in which case a mentor doesn't have dojo-admin by default, which is the current restriction for the API call letting us loading all users of a dojos.
Ticketing-admin <perm> can get this data, but only event by event, for users which are part of the events.

Reply:
tldr : the API is not done for stats purpose

Reply:
cc @conoramurphy (please tell me if I'm wrong @DanielBrierton )